tests/bluechi_test/machine.py

```diff
@@ -276,10 +276,22 @@ def gather_coverage(self, data_coverage_dir: str) -> None:
         coverage_file = f"{BluechiMachine.gcda_file_location}/coverage-{self.name}.info"
 
         LOGGER.info(f"Generating info file '{coverage_file}' started")
-        bluechi_version = get_env_value("BLUECHI_VERSION", "unknown")
+
+        bluechi_build_dir = ".*/rpmbuild/BUILD/bluechi[^/]*"
```

Comment:
If I understand this correctly, then the build dir pattern is kind of "universal" - so we can remove it here, don't need to pass it to the script and instead hardcode it there. For example instead of
```bash
BLUECHI_BUILD_DIR=${2:-/github/home/rpmbuild/BUILD/bluechi-0.1.0}
```
we can have 
```bash
BLUECHI_BUILD_DIR_PATTERN=".*/rpmbuild/BUILD/bluechi[^/]*"
```
and directly use this in the `substitute` option. WDYT? @ArtiomDivak

Reply:
You are right. Changed Thank you